app/Handlers/Commands/Metric/AddMetricPointCommandHandler.php

```diff
@@ -36,7 +36,7 @@ public function handle(AddMetricPointCommand $command)
         ];
 
         if ($createdAt) {
-            $data['created_at'] = Carbon::createFromFormat('U', $createdAt)->format('Y-m-d H:i:s');
+            $data['created_at'] = Carbon::createFromFormat('U', $createdAt, config('cachet.timezone'))->format('Y-m-d H:i:s');
```

Comment:
i think we can get away with global functions in controllers, views, and middleware, but we shouldn't use them anywhere else at all